test/hotspot/gtest/runtime/test_committed_virtualmemory.cpp

```diff
@@ -214,3 +214,70 @@ TEST_VM(CommittedVirtualMemoryTracker, test_committed_virtualmemory_region) {
   }
 
 }
+
+#if !defined(_WINDOWS) && !defined(_AIX)
+TEST_VM(CommittedVirtualMemory, test_full_committed_in_range){
```

Comment:
These two tests are almost the same. Please factor out into a single file static test function, with num pages as arguments, and call that from both TESTs